easycwmp: fix code typo on openwrt action script

Remove a stray "$" from the firmware download processing (tar.gz/tar.bz2 code
paths).  This code path is used to override/deploy new data files in
/etc/config or in /etc.

Upstream-report: http://support.easycwmp.org/view.php?id=303

--- a/ext/openwrt/scripts/easycwmp.sh
+++ b/ext/openwrt/scripts/easycwmp.sh
@@ -296,7 +296,7 @@ handle_action() {
 					tar -zxf $dwfile -C $DOWNLOAD_DIR >/dev/null 2>&1
 					fault_code="$?"
 					if [ "$fault_code" = "0" ]; then
-						if [ -d $ $DOWNLOAD_DIR/config/ ]; then
+						if [ -d $DOWNLOAD_DIR/config/ ]; then
 							cp -R $DOWNLOAD_DIR/config/* /etc/config/
 						else
 							cp -R $DOWNLOAD_DIR/* /
@@ -306,7 +306,7 @@ handle_action() {
 					tar -jxf $dwfile -C $DOWNLOAD_DIR >/dev/null 2>&1
 					fault_code="$?"
 					if [ "$fault_code" = "0" ]; then
-						if [ -d $ $DOWNLOAD_DIR/config/ ]; then
+						if [ -d $DOWNLOAD_DIR/config/ ]; then
 							cp -R $DOWNLOAD_DIR/config/* /etc/config/
 						else
 							cp -R $DOWNLOAD_DIR/* /
